content/docs/1.0.0/tutorial/create-your-first-microservice-javascript.md

```diff
@@ -326,7 +351,7 @@ We now to need to package up our code as a Docker image, so that we can deploy i
 Like all build scripts, Dockerfiles can become a bit complex for advanced applications, but fortunately for our simple microservice, ours is pretty simple:
```

Comment:
Like all build scripts, Dockerfiles can become a bit complex for advanced applications, but fortunately**, our microservice is fairly simple**: